docs/dev-guide/conventions/sync-v-async.rst

```diff
@@ -16,9 +16,11 @@ A success response indicates no conflicts were detected and the REST call
 executed. This is what is typically expected from a REST API call.
 
 A postponed response indicates that some portion of the command has been
-queued to execute asynchronously.  In this case a :ref:`call_report` will be returned
-with the results of the synchronously executed portion of the command, if there are any,
-and a list of the tasks that have been spawned to complete the work in the future.
+queued to execute asynchronously.  In this case a :ref:`call_report` or a :ref:`group_call_report`
+will be returned with the results of the synchronously executed portion of the command. If there
+are any tasks that were dispatched to execute serially, a :ref:`call_report` will contain a list
+of spawned tasks. If a lot of tasks were dispatched to execute in parallel, a
+:ref:`group_call_report` is returned.
```

Comment:
When a task queues more tasks that are "next steps", like a publish after a sync, or unbind consumers after a repo delete, those are spawned tasks. The spawned tasks are not necessarily related to each other, now will a client necessarily want to track them together.

When some number of related tasks are queued, and it is useful to track the state of all of them together, that is when a task group is used.

I would not try to make a serial vs. parallel explanation.